scripts/release/createPWTRelease.ts

```diff
@@ -1,8 +1,9 @@
-import { getPackageInfo, PackageInfo } from "../../packages/tools/release-utils-internal";
-import gh from "../../packages/tools/release-utils-internal/src/github";
+import { getPackageInfo, PackageInfo } from "../../packages/tools/release-utils-internal/dist";
+import gh from "../../packages/tools/release-utils-internal/dist/github";
 import { join } from "path";
-import { addRemoteWithAuthentication } from "../../packages/tools/release-utils-internal/src/git";
+import { addRemoteWithAuthentication } from "../../packages/tools/release-utils-internal/dist/git";
 import { execShellCommand } from "../../packages/tools/release-utils-internal/dist/shell";
+import { WidgetChangelogFileWrapper } from "../../packages/tools/release-utils-internal/dist/changelog-parser";
```

Comment:
Is it possible to add "files" or "main" to package.json to not pont to `dist` folder?